gulpfile.js

```diff
@@ -43,6 +43,9 @@ var argv = yargs
         .option('clear_datastore', {
           describe: 'Whether to clear all data storage'
         })
+        .option('prod_env', {
```

Comment:
Maybe instead of ``prod_env`` we could do ``use_minification``? I worry about the inconsistency here. It's sort of saying that "running like production = minified scripts," but that's not quite true. It's probably more consistent just to keep the idea of using minifcation throughout the change. WDYT?